tox.ini

```diff
@@ -304,6 +305,7 @@ commands =
     opentracer_gevent: pytest {posargs} tests/opentracer/test_tracer_gevent.py
 # integration tests
     integration: pytest {posargs} tests/test_integration.py
+    metrics: pytest {posargs} tests/test_metrics.py
```

Comment:
These will get included in `*-tracer` test runs as well, we need to omit from the `*-tracer` command.